tests/e2e/default/enketo/edit-report-with-attachment.wdio-spec.js

```diff
@@ -89,10 +85,6 @@ describe('Edit report with attachmnet', () => {
   });
 
   it('should save edits', async () => {
-    reportDoc._id = uuid();
-    await utils.saveDoc(reportDoc);
-
-    await commonElements.goToReports();
```

Comment:
How do you ensure you are on the reports tab?

Reply:
See the previous answer 😊 

The call to `await commonElements.goToReports();` was already includded in the the method `editReport`, so it was a duplicated action.